### PR TITLE
As user/edit traveler info

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java [hunk 1]
```diff
@@ -55,4 +59,52 @@ public boolean createUser(TUser tUser) {
...
+				System.out.println(ex.getMessage());
+			}
+			
+			if (user != null){
+				Traveler traveler = new Traveler(tTraveler.getNickname(),
+												tTraveler.getRating(),
+												tTraveler.getDescription(),
+												tTraveler.getHost(),
+												tTraveler.getListOfCountries(),
+												tTraveler.getListOfKnowledges());
```
Comment (adridean) [inline HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java]: Si es viajero, modifica la info; si no es viajero, crea un nuevo viajero con esa info. Ese if esta incompleto

Comment (ivanalba1) [reply HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java]: +1

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		
+		boolean result =  false;
+		
+		try {
+			EntityManagerFactory emf = Persistence.createEntityManagerFactory("HostAbroad");
+			EntityManager em = emf.createEntityManager();
+			EntityTransaction tr = em.getTransaction();
+			tr.begin();
+
+			String consulta = "SELECT user FROM User user WHERE user.nickname = :nickname";
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java:30]: This Select is not working>run your code then put it in pull request

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+	@Override
+	public boolean editTravelerInformation(TTraveler tTraveler) {
+		boolean result =  false;
+		try {
+			EntityManagerFactory emf = Persistence.createEntityManagerFactory("HostAbroad");
+			EntityManager em = emf.createEntityManager();
+			EntityTransaction tr = em.getTransaction();
+			tr.begin();
+			
+			String consulta = "SELECT user FROM User user WHERE user.nickname = :nickname";
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java]: Same as the previous.

File: HostAbroad/src/main/java/com/business/CountriesEnum.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	Australia,
+	Austria,
+	Brazil,
+	Bulgaria,
+	Canada,
+	Chile,
+	China,
+	Colombia,
+	CostaRica,
+	Denmark,
```
Comment (GasanNazer) [inline HostAbroad/src/main/java/com/business/CountriesEnum.java:14]: Pls remove all of the other files. Just commit the classes you've changed it this task.

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+												tTraveler.getHost(),
+												tTraveler.getListOfCountries(),
+												tTraveler.getListOfKnowledges());
+				if(!user.isTraveler()){
+					user.setTraveler(true);
+					em.persist(user);
+				}
+				em.persist(traveler);
+				
+			}
```
Comment (ivanalba1) [inline HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java:86]: Falta hacer commit

File: HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java (hunk 1 above)
Comment (ivanalba1) [reply HostAbroad/src/main/java/com/business/ASUser/ASUserImp.java]: Recuerda cambiarlo que tenemos que cerrar la tarea

**Pull request merged**